### DISCUSSION TITLE
kibrisraporu.com: detection

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uAssets#24495] [Q&A]
`https://www.kibrisraporu.com/`
https://i.ibb.co/F0gFKp2/kpq.png

A problem that occurs when using the Brave browser. It is not detected in Chrome browser.
Can you please suggest user filter to bypass detection

**Comment by garry-ut99** [comment]
It doesn't occur if you only use uBlock alone, it's caused by Brave's build-in ad blocker, disable it and don't use it at the same time with uBlock, as a single ad blocker is sufficient:
- https://github.com/uBlockOrigin/uBOL-home/discussions/126
- https://brave.com/learn/turn-off-ad-blockers/

If you don't use uBlock in Brave, and only use built-in ad blocker, then it's wrong place, report your issue to Brave forum:
- https://community.brave.com/c/support-and-troubleshooting/ad-blocking/78

(Unless the issue occurs for you even with uBlock alone.)
